src/core/room.ts

```diff
@@ -239,6 +247,15 @@ export class DefaultRoom implements Room {
       manager.mergeOptions(DefaultPresence.channelName(this._roomId), DefaultPresence.channelOptionMerger(options));
     }
 
+    manager.mergeOptions(messagesChannelName(this._roomId), (opts) => {
+      if (opts.modes) {
+        opts.modes.push('ANNOTATION_PUBLISH');
+      } else {
+        opts.modes = ['publish', 'subscribe', 'presence_subscribe', 'presence', 'ANNOTATION_PUBLISH'];
```

Comment:
Is the difference in casing intentional here?

Reply:
I think we have some issue with this, actually.

In tests, the casing is always uppercase, so I'm assuming this never runs (only the first condition).

Previously I put it lowercase everywhere, and in test the values look all uppercase except the new one I set.

Maybe change them all to upper? Or I can dig some more to see where these come from?

Reply:
Lets dig more to see where these come from - though I believe that if we look at the underlying ably-js types, they should be lowercase?